app/ide-desktop/lib/dashboard/src/modals/TermsOfServiceModal.tsx

```diff
@@ -83,43 +94,32 @@ export function TermsOfServiceModal() {
           id="terms-of-service-modal"
         >
           <ariaComponents.Form
+            schema={formSchema}
+            defaultValues={{ agree: false, hash: latestVersionHash }}
             testId="terms-of-service-form"
-            schema={ariaComponents.Form.schema.object({
-              agree: ariaComponents.Form.schema
-                .boolean()
-                // we accept only true
-                .refine(value => value, getText('licenseAgreementCheckboxError')),
-            })}
-            onSubmit={() => {
-              localStorage.set('termsOfService', { versionHash: latestVersionHash })
+            onSubmit={({ hash }) => {
+              localStorage.set('termsOfService', { versionHash: hash })
             }}
           >
             {({ register, formState }) => {
               const agreeError = formState.errors.agree
-              const hasError = formState.errors.agree != null
-
-              const checkboxRegister = register('agree')
+              const hasError = agreeError != null
 
               return (
                 <>
-                  <div className="pb-6 pt-2">
+                  <div>
                     <div className="mb-1">
                       <div className="flex items-center gap-1.5 text-sm">
-                        <div className="mt-0">
-                          <aria.Input
-                            type="checkbox"
-                            className={twMerge.twMerge(
-                              `flex size-4 cursor-pointer overflow-clip rounded-lg border border-primary outline-primary focus-visible:outline focus-visible:outline-2 ${hasError ? 'border-red-700 text-red-500 outline-red-500' : ''}`
-                            )}
-                            id={checkboxId}
-                            aria-invalid={hasError}
-                            {...checkboxRegister}
-                            onInput={event => {
-                              void checkboxRegister.onChange(event)
-                            }}
-                            data-testid="terms-of-service-checkbox"
-                          />
-                        </div>
+                        <aria.Input
+                          type="checkbox"
+                          className={twMerge.twMerge(
```

Comment:
bad `twMerge`

Reply:
Sorry?

Reply:
this `twMerge` contains 1 parameter which is a template literal containing a ternary

Reply:
Sorry, I'm not seeing a problem here.

Reply:
For twMerge it doesn't matter if it's a single string or multiple strings. it merges classes, not strings